readme.md

```diff
@@ -22,6 +22,8 @@ Work in the ```src/``` folder. You can manually transpile your ES6 syntax back t
 $ git clone https://github.com/NREL/eere-ie-projects-map.git
 $ npm install
 ```
+If you have issues installing npm, see https://github.nrel.gov/TADA/nrel-certs.
```

Comment:
Can you change "installing" to "running" here? Or "installing packages with". Basically we're assuming npm is already installed.